Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -966,7 +962,11 @@ public List<IMixedRealityService> GetActiveServices(Type type, string serviceNam
         private void InitializeAllServices()
         {
             //If the Mixed Reality Toolkit is not configured, stop.
-            if (activeProfile == null) { return; }
+            if (ActiveProfile == null)
+            {
+                DebugUtilities.LogCriticalError("Unable to intialize services. There is no active profile configured.");
```

Comment:
Fairly certain we could never get this far anyway. But I suppose it doesn't hurt to have it.